src/HDF5.jl

```diff
@@ -25,7 +25,7 @@ export
     g_create, g_open, get_access_properties, get_create_properties,
     get_chunk, get_datasets,
     h5open, h5read, h5rewrite, h5writeattr, h5readattr, h5write,
-    has, iscontiguous, ishdf5, ismmappable, name,
```

Comment:
oops

Reply:
Hah, within 5 second of each other :grin: